tailwind.config.js

```diff
@@ -410,7 +435,7 @@ module.exports = {
         '15px': '0.938rem',
         '18px': '1.125rem',
         '19px': '1.1875',
-        '30px': '2rem',
+        '30px': '1.875',
```

Comment:
No unit specified here, and now we can use px as unit
Some styles may be broken by this, so can't approve

Reply:
Updated